app/views/admin/admin_users/_user_history.html.erb

```diff
@@ -13,17 +13,27 @@
       <% unless @log_items.blank?
         @log_items.each do |item| %>
         <tr>
-          <td><%= item.created_at %></td>
+          <td><%= item.created_at %> ( <abbr class="timeago" title="<%= item.created_at.iso8601 =%>"><%= item.created_at.iso8601  =%></abbr> )</td>
```

Comment:
Can you give me some example output for this? That is, will it look like one of these? 
`<td>2014-11-03 22:53:47 -0500 (<abbr class="timeago" title="2 months ago">2 months ago</abbr>)</td>`

*`<td>2014-11-03 22:53:47 -0500 (<abbr class="timeago" title="2 months ago">2 months ago</abbr>)</td>`

Also, I think you have might have too many `=` signs -- it's usually `<%= item.created_at.iso8601 %>` -- but if the second `=` is required for the iso method to work, just ignore me.